src/tests/ftest/slurm_setup.py

```diff
@@ -55,6 +57,28 @@
     "grep -v \"^#\\w\" /etc/slurm/slurm.conf"]
 
 
+def create_epilog_script(args):
+    """Create epilog script to run after each job.
+
+    Args:
+        args (Namespace): command line arguments
+
+     Returns:
+        returns 0 if command pass; 1 otherwise
+
+    """
+    sudo = "sudo" if args.sudo else ""
+    with open(EPILOG_FILE, 'w') as script_file:
+        script_file.write("#!/bin/bash\n#\n")
+        script_file.write("/usr/bin/bash -c 'pkill --signal 9 dfuse'\n")
+        script_file.write("/usr/bin/bash -c 'for dir in $(find /tmp/daos_dfuse);"
+                          "do fusermount3 -uz $dir;rm -rf $dir; done'\n")
+        script_file.write("exit 0\n")
+        script_file.close()
```

Comment:
There's no need to close the file when using `with open`. It's done automatically
```suggestion
```